includes/Classifai/Services/LanguageProcessing.php

```diff
@@ -167,6 +186,34 @@ public function generate_post_excerpt( WP_REST_Request $request ) {
 		return rest_ensure_response( $provider->rest_endpoint_callback( $post_id, 'excerpt' ) );
 	}
 
+	/**
+	 * Generates text to speech for a post using REST.
+	 *
+	 * @param WP_REST_Request $request Full data about the request.
+	 * @return boolean
```

Comment:
Right now this always returns an `array`, not a `boolean`. But if we utilize `rest_ensure_response` here, we should change this to:

```suggestion
	 * @return \WP_REST_Response|WP_Error
```